Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -121,27 +123,13 @@ public static bool AddPlayer(NetworkConnection readyConn)
             return true;
         }
 
+        // Deprecated 5/2/2020
         /// <summary>
-        /// Removes the player from the game.
+        /// Obsolete: Removed as a security risk
         /// </summary>
         /// <returns>True if succcessful</returns>
-        public static bool RemovePlayer()
-        {
-            if (logger.LogEnabled()) logger.Log("ClientScene.RemovePlayer() called with connection [" + readyConnection + "]");
-
-            if (readyConnection.identity != null)
-            {
-                readyConnection.Send(new RemovePlayerMessage());
-
-                Object.Destroy(readyConnection.identity.gameObject);
-
-                readyConnection.identity = null;
-                localPlayer = null;
-
-                return true;
-            }
-            return false;
-        }
+        [EditorBrowsable(EditorBrowsableState.Never), Obsolete("Removed as a security risk", true)]
```

Comment:
the other pr was just merged.
so it would be good if the obsolete message tells people to call server.removeplayer instead.
also, maybe we should keep the functionality in here for now and give people some time to adjust their code?
that's what obsolete is for anyway